metaseq/cli/train.py

```diff
@@ -288,7 +299,23 @@ def train(
 
         return valid_losses, should_stop
 
+    skip_batches = None
+    if len(cfg.dataset.skip_batches) > 0:
+        skip_batches = get_skip_batches(cfg.dataset.skip_batches)
     for i, samples in enumerate(progress):
+        current_step = trainer.get_num_updates() + 1
+        if (
+            skip_batches is not None
+            and current_step in skip_batches
+            and skip_batches[current_step] > 0
+        ):
+            skip_batches[current_step] -= 1
```

Comment:
Hmm not sure this works - skip_batches is keyed by starting batch index, so if there's more than one batch to skip (passed in range), then by this logic I think we will only ever skip one batch in that range...

Reply:
We can either add a new logged metric for each step that shows the current shard and shard iteration of a batch, which would give us an absolute numbering, and skip based on that.

Or use relative numbering with the num_updates that we already have. And we can always afterwards convert the skip_batches list into a absolute list for accounting